src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { RWD_MODES } from '../../../redux/initialState';
+
 import styles from './NewFurniture.module.scss';
```

Comment:
Żeby od razu załatwić sprawę rozjeżdżających się kropek na mobilce, zaimportuj nasze mixiny od mediaQuery w NewFurniture.module.scss i w panelBar pod margin-bottom daj:

```
@include for-phone {
    margin-bottom: 80px;
}
```

a w .dots pod top: 0 daj:

      @include for-phone {
        top: 60px;
        text-align: center;
      }

Będzie to wtedy ładnie wyglądało ;)

Reply:
Zmieniłem nieco style żeby były wyśrodkowane ładniej na tabletach.